Utilities/MCStepLogger/include/MCStepLogger/mcanalysishelper.h

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
I think the convention for file names is `MCAnalysisHelpers.h` rather than `mcanalysishelper...`